packages/edit-post/src/components/sidebar/post-status/index.js

```diff
@@ -45,13 +45,18 @@ function PostStatus( { isOpened, onTogglePanel } ) {
 }
 
 export default compose( [
-	withSelect( ( select ) => ( {
-		isOpened: select( 'core/edit-post' ).isEditorPanelOpened( PANEL_NAME ),
-	} ) ),
+	withSelect( ( select ) => {
+		const { isEditorPanelEnabled, isEditorPanelOpened } = select( 'core/edit-post' );
+		return {
+			isEnabled: isEditorPanelEnabled( PANEL_NAME ),
+			isOpened: isEditorPanelOpened( PANEL_NAME ),
+		};
+	} ),
 	withDispatch( ( dispatch ) => ( {
 		onTogglePanel() {
 			return dispatch( 'core/edit-post' ).toggleEditorPanelOpened( PANEL_NAME );
 		},
 	} ) ),
+	ifCondition( ( { isEnabled } ) => isEnabled ),
```

Comment:
It could even be moved before withDispatch. Probably we won't gain anything perceivable but you know bail as early as possible :P

Reply:
Done ;)